project/src/main/webapp/event-details.js

```diff
@@ -1,12 +1,27 @@
 window.onload = async function onLoad() {
   getEventDetails();
   const eventHost = await getEventHost();
+  configureRegisterAndEditButtons(eventHost);
   const loginStatus = await getLoginStatus();
   populateVolunteeringOpportunitiesUI(eventHost, loginStatus);
   showCreateOpportunityLink(eventHost, loginStatus);
   setSignupAction();
 };
 
+/** 
+ * Adds the edit button for the host and removes the register button
```

Comment:
If this removes the register button for the host, in what case would you get the alert for 'you cant register bc you are the host'? is it if the user manually navigates to the url?

Reply:
They're 2 separate pages - the alert shows up when you click on register in the feed, and once you click into event details, it hides that register button